### ISSUE TITLE
godaddy: can't delete records HTTPError: 422 Client Error: Unprocessable Entity for url

### ISSUE THREAD

**Issue by zeule** [AnalogJ/lexicon#1019]
`$ lexicon godaddy --auth-key <key> --auth-secret <secret> delete <domain> TXT --name '<name>'` results in:
```
Traceback (most recent call last):
  File "/usr/lib/python-exec/python3.9/lexicon", line 33, in <module>
    sys.exit(load_entry_point('dns-lexicon==3.8.3', 'console_scripts', 'lexicon')())
  File "/usr/lib/python3.9/site-packages/lexicon/cli.py", line 131, in main
    results = client.execute()
  File "/usr/lib/python3.9/site-packages/lexicon/client.py", line 97, in execute
    return self.provider.delete_record(identifier, record_type, name, content)
  File "/usr/lib/python3.9/site-packages/lexicon/providers/base.py", line 118, in delete_record
    return self._delete_record(
  File "/usr/lib/python3.9/site-packages/lexicon/providers/godaddy.py", line 261, in _delete_record
    self._put(f"/domains/{domain}/records", filtered_records)
  File "/usr/lib/python3.9/site-packages/lexicon/providers/base.py", line 189, in _put
    return self._request("PUT", url, data=data, query_params=query_params)
  File "/usr/lib/python3.9/site-packages/lexicon/providers/godaddy.py", line 328, in _request
    result.raise_for_status()
  File "/usr/lib/python3.9/site-packages/requests/models.py", line 953, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 422 Client Error: Unprocessable Entity for url: https://api.godaddy.com/v1/domains/<domain>/records
```

**Comment by zeule** [comment]
The update action runs into the same error.

**Comment by zeule** [comment]
Fixed in 3.8.5.
